behaviour/graql/language/define.feature

```diff
@@ -636,112 +598,61 @@ Feature: Graql Define Query
       | FAA |
 
 
-  Scenario: define abstract relation type creates a type
+  Scenario: define abstract relation type with no roleplayers creates a type
```

Comment:
maybe call this one "is valid" at the end, instead of "creates a type", is more descriptive

Reply:
To be honest, quite a lot of the tests could be renamed to include 'is valid', I just find the words a bit redundant. I prefer to maintain some consistency by specifying:
1- the action "define abstract relation type with no roleplayers"
2- the outcome "creates a type"

But in this case I suppose 'creates a type' is no better, giving across the meaning of 'creating a type creates a type'. It's a tough one really. But I definitely have an aversion to writing "is valid" as the outcome - to me that's not a valid outcome.

I think I'll go with `define relation type with no roleplayers is possible when marked as abstract`, emphasising that the outcome is you've defined a relation type with no roleplayers.

Reply:
Or even `is valid when marked as abstract`, heh

Reply:
lets discuss this on a call

Reply:
Renamed to `a relation type can be defined with no roleplayers when it is marked as abstract`